import exceptionsLib.*

program exceptions {
val a = new A()

var i = 0
try {
	a.m1()
	self.println("No exception raised")
}
catch e : MyException
	e.printStackTrace()

then always
	i = i + 1

if (i > 0) {
	self.println("i is positive")
}
else {
	self.println("i is zero !")
}
	

self.println("End of program")

}